sdk/python/feast/client.py

```diff
@@ -382,6 +386,25 @@ def version(self):
 
         return result
 
+    def _configure_telemetry(self):
+        telemetry_filepath = join(expanduser("~"), ".feast", "telemetry")
+        self._telemetry_enabled = self._config.get(opt.TELEMETRY, "True") == "True"
+        if self._telemetry_enabled:
+            self._telemetry_counter = {"get_online_features": 0}
+            if os.path.exists(telemetry_filepath):
+                with open(telemetry_filepath, "r") as f:
+                    self._telemetry_id = f.read()
+            else:
+                self._telemetry_id = str(uuid.uuid4())
```

Comment:
So even if they delete the file, we will continue to recreate it?

Reply:
yes, I don't see a way around that because the file won't exist the very first time. I clarified the docs a bit